spring-cloud-generator/src/test/java/com/google/cloud/generator/spring/utils/TestUtils.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.generator.spring.utils;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+// Renamed from Utils
+public class TestUtils {
```

Comment:
That’s a good point, I think we could benefit from cleaning this up at the gapic-generator-java source as well, since it gets confusing with classes that are meant to be tests for utils vs. utils for tests 🤯 

I’ve started a [WIP PR](https://github.com/googleapis/gapic-generator-java/pull/1648) in gapic-generator-java to publish a testlib jar that can hopefully get rid of the need for this file duplication altogether. In the meanwhile, will rename this helper class to something less ambiguous here.

Reply:
Renamed to `GoldenFileWriter`.